### ISSUE TITLE
[Bug] | Users re-appear in the users list even after being successfully deleted

### ISSUE THREAD

**Issue by 0penSourcerer** [wizarrrr/wizarr#492]
### Are you using the latest release?

- [X] Yes, I am using the latest release.

### Have you checked a bug report doesn't already exist?

- [X] Yes, I have checked the existing bug reports.

### What happened?

After deleting a user...
<img width="1158" alt="Screenshot 2024-11-18 at 10 16 35 PM" src="https://github.com/user-attachments/assets/10c298b4-a35f-4968-be79-dd9a0080984b">

...I get a notification on the screen that the user has been successfully deleted, but after using the "Scan Users" tool...
<img width="1159" alt="Screenshot 2024-11-18 at 10 17 07 PM" src="https://github.com/user-attachments/assets/637a43de-46b2-42c6-a08f-e8ee112b5af3">

...the user will be back in the list. The automated Scan Users task will also cause the user to re-appear when it has run.
<img width="1170" alt="Screenshot 2024-11-18 at 10 17 40 PM" src="https://github.com/user-attachments/assets/1385386a-1a4e-4a2c-a3e0-93186159a900">

### Virtualisation

Docker

### What operating system are you seeing the problem on?

Linux

### What browsers are you seeing the problem on?

Chrome

### Project Location

Local

### Bug Description

Deleting users doesn't seem to be working properly, as users re-appear on the User page after a supposed successful deletion.

### Steps to Reproduce

1. be logged into the Admin Dashboard
2. navigate to the Users page
3. delete the intended user
4. wait for the Scan Users task to run, or run the task manually using the "Scan Users" button.
5. the user will reappear on the Users page.

**Comment by zbejas** [comment]
I would like to add that when that removed user is readded, the user won't get an expiration timer set on them.

I don't know if it has anything to do with the [user api](https://plex.tv/api/users), or it just fails to cause the user is already "added". It would be nice if the `Server` tags were checked since the new friend system messed everything up.

**Comment by t-abraham** [comment]
I think it's more into a Plex and old and new api issue...

Perhaps this thread will be helpful for the developers. Comment 4 talks about it

[Removed access to my PMS for some users but they still show in XML file?](https://forums.plex.tv/t/removed-access-to-my-pms-for-some-users-but-they-still-show-in-xml-file/871636/8)

**Comment by EarthSalamander42** [comment]
Is it not related to the fact they're still friends of the server owner although not having an access to the server anymore?

**Comment by 0penSourcerer** [comment]
> Is it not related to the fact they're still friends of the server owner although not having an access to the server anymore?

No. Many of these users I was never "friends" with in the Plex network...they were created solely as tests to test Wizarr with. What you're describing has zero relevance to the issue at hand.

**Comment by impressiveaustin** [comment]
I have the same issue. Did you ever get it fixed? Is it a cache thing on Plex's end?
